devicetypes/smartthings/zwave-basic-window-shade.src/zwave-basic-window-shade.groovy

```diff
@@ -0,0 +1,126 @@
+/**
+ *  Copyright 2019 SRPOL
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+metadata {
+	definition (name: "Z-Wave Basic Window Shade", namespace: "smartthings", author: "SmartThings") {
+		capability "Stateless Curtain Power Button"
+		capability "Configuration"
+		capability "Actuator"
+		capability "Health Check"
+
+		command "open"
```

Comment:
@dkirker The `Stateless Curtain Power Button` capability is only proposed, but these commands are part of it, so they shouldn't need to be defined separately, right?

Reply:
Based on the commented-out code below @PKacprowiczS I'm guessing it didn't work without them, though?

Reply:
So, it appears that `Stateless Curtain Power Button` has the one command `setButton` which takes one of `["open","close","pause"]`.

Reply:
ah, makes sense